installation/local-hosting-vps/macos.md

```diff
@@ -4,8 +4,70 @@ description: Deploy Modmail on a MacOS machine.
 
 # MacOS
 
-TODO
-
 ## Prerequisites
 
-## Updating
+1. Minimum 2GB of RAM
+2. At least 2GB available disk space.
+3. Support MacOS version:
+   - MacOS 14 Sonoma (Tested)
+
+## Dependencies
+
+We will be using the following dependencies:
+
+* Python 3.10
+* Tools: `git`, `homebrew`
+* Additional Modmail requirements: `cairo`, `libxml2`, `libxslt`, `libffi`
+
+{% hint style="info" %}
+All code blocks should be executed in bash and line by line unless specified otherwise.
```

Comment:
Not what I was referring to, while thats a good catch I missed it too, I meant the block that currently says 
> All code blocks should be executed in bash and line by line unless specified otherwise.


which should more nearly reflect
> All code blocks should be executed in bash or zsh and line by line unless specified otherwise.